buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -75,7 +75,7 @@
  *   OPENJDK<version>_SHA_<platform>: String - the last commit SHA
  */
 
-CURRENT_RELEASES = ['8', '11', '12', 'next']
+CURRENT_RELEASES = ['8', '11', '12', '13', 'next']
```

Comment:
I don't want to add 13 to the nightly build, I don't think we have the machine resources for all platforms. Plus the system tests will always fail until the problems are resolved.

Reply:
The nightly build (and any build) specifies what versions it wants to build so we'd have to add the boolean param to the nightly config in order to enable it.